src/api/queries/sykmeldt/narmesteLedereQueriesSM.ts

```diff
@@ -1,7 +1,7 @@
 import { useApiBasePath } from "../../../hooks/routeHooks";
 import { get } from "../../axios/axios";
 import { useSykmeldtFnr } from "./sykmeldingerQueriesSM";
-import { NarmesteLeder } from "../../../schema/narmestelederSchema";
+import { NarmesteLederDTO } from "../../../schema/narmestelederSchema";
```

Comment:
Burde schema være i node/server?

Reply:
Ja, det burde det men den er ikke typet opp enda :P Var litt redd for scope creep men skal se på det i en annen PR